Web/starter_project_barbies/components/blog/RichTextEditor.tsx

```diff
@@ -0,0 +1,38 @@
+/* This component is done by using react-quill rich text editor */
+import dynamic from 'next/dynamic';
+import 'react-quill/dist/quill.snow.css';  
+
+const QuillEditor = dynamic(
```

Comment:
i am assuming this is returned as a component (which is the only reason i would allow the PascalCase)

Reply:
i am not sure why you have to avoid the ssr tho

Reply:
Since the Quill editor is designed to work in the browser environment, it may not function properly if it is rendered on the server. That's why I set "ssr : false", to remove errors that can occur when server-side rendering a component that relies on browser-specific APIs or features.